splinter/driver/djangoclient.py

```diff
@@ -8,7 +8,8 @@
 import os.path
 import re
 import sys
-import urlparse
+import six
```

Comment:
It is not a problem. But it is needed to add the `six` dependencie in setup.py: https://github.com/kidosoft/splinter/blob/master/setup.py#L28